packages/hunspell.rb

```diff
@@ -2,37 +2,64 @@
 
 class Hunspell < Package
   description 'Hunspell is a spell checker and morphological analyzer library'
-  homepage 'http://hunspell.github.io/'
+  homepage 'https://hunspell.github.io'
   version '1.7.0'
   license 'MPL-1.1, GPL-2 and LGPL-2.1'
   compatibility 'all'
-  source_url 'https://github.com/hunspell/hunspell/archive/v1.7.0.tar.gz'
-  source_sha256 'bb27b86eb910a8285407cf3ca33b62643a02798cf2eef468c0a74f6c3ee6bc8a'
 
   is_fake
 
-  if ARGV[0] == 'install'
-    puts
-    puts 'Enter your preferred language:'
-    puts '1 = American English'
-    puts '2 = Français'
-    puts '3 = Español'
-    puts '0 = Cancel'
-
-    while version = $stdin.gets.chomp
-      case version
-      when '1'
-        depends_on 'hunspell_en_us'
-        break
-      when '2'
-        depends_on 'hunspell_fr_fr'
-        break
-      when '3'
-        depends_on 'hunspell_es_any'
-        break
-      when '0'
-        abort
+  if ARGV.include?('install') # fix when 'install' isn't in ARGV[0] (like `crew -k install`)
+    @timeout = 10
+
+    puts <<~EOT
+
+      Enter your preferred language:
+      (The default, American English, will be selected in #{@timeout} seconds if there is no input.)
+
+        1 = American English
+        2 = Français
+        3 = Español
+        0 = Cancel
+
+    EOT
+
+    if $stdin.isatty # check if stdin is a terminal
+      @stdin_thread = Thread.new do
+        @user_input = $stdin.getc.to_i
+
+        Thread.kill(@countdown_thread) # stop countdown when user input
       end
+
+      @countdown_thread = Thread.new do
+        (0..@timeout).each do |i|
+          remaining_time = @timeout - i
+          print "\r#{remaining_time} second(s) left: "
```

Comment:
Why `\r` and not `\n`?  Remove `:`?

Reply:
@supechicken?

Reply:
The `\r` char represents to "reprint the current line" (see `downloader.rb` for more info), and `:` is for prompting user to input